lib/matplotlib/backends/backend_ps.py

```diff
@@ -362,8 +362,7 @@ def create_hatch(self, hatch):
      /PaintProc {{
         pop
         {linewidth:g} setlinewidth
-{self._convert_path(
-    Path.hatch(hatch), Affine2D().scale(sidelen), simplify=False)}
+{self._convert_path(Path.hatch(hatch), Affine2D().scale(sidelen), simplify=False)}
```

Comment:
NB: This line-change isn't directly related to the pull request, but was flagged by `pre-commit.ci` as an `E122` (indentation) problem.

Reply:
This was resolved in mainline by #28136.